tools/api-markdown-documenter/.gitignore

```diff
@@ -1,5 +1,5 @@
 # Build output
-dist
+lib
```

Comment:
Probably fine to remove `dist` as `dist` (and `lib`) ignores are set in root .gitignore.
Otherwise, I recommend holding onto the ignore for a while as otherwise people who have old builds will suddenly have checked out files. (I've seen files get checked in after such a change.)